package.json

```diff
@@ -38,7 +40,11 @@
     "yaml": "^2.0.1"
   },
   "devDependencies": {
-    "c8": "^7.11.2"
+    "c8": "^7.11.2",
+    "simple-git-hooks": "^2.7.0"
```

Comment:
Let’s not install hooks. This is unfortunately not supposed on one of my internal uses.

Reply:
Ok.